DNN Platform/Providers/CachingProviders/DotNetNuke.Providers.Caching.SimpleWebFarmCachingProvider/SimpleWebFarmCachingProvider.dnn

```diff
@@ -1,6 +1,6 @@
 ﻿<dotnetnuke type="Package" version="5.0">
   <packages>
-    <package name="DotNetNuke.Providers.Caching.SimpleWebFarmCachingProvider" type="Provider" isSystem="true">
+    <package name="DotNetNuke.Providers.Caching.SimpleWebFarmCachingProvider" type="Provider" isSystem="true" version="09.04.04">
```

Comment:
Should this be `version="09.05.00"`

Reply:
It should be, but currently all versions are on 9.4.4 in the repo. So just sticking with what is there.

Reply:
In any case: it gets changed upon build

Reply:
Okay, thanks for the explanation.  Marking resolved, approving and merging.